### ISSUE TITLE
Move this repo to thelounge org

### ISSUE THREAD

**Issue by xPaw** [astorije/ansible-lounge#12]
Seems like a good idea

**Comment by astorije** [comment]
Hey @xPaw sorry I totally forgot about this. I agree and I'm happy to transfer the ownership to you or whatnot. Anything you'd like me to do there?

**Comment by xPaw** [comment]
Considering it hasn't been updated in a while, I don't know if it still works with the current state of things (I haven't looked at lounge in a while myself).

**Comment by astorije** [comment]
Fair enough. If anyone else sees this and wants to move it to the official org, I'm super open to it!
